src/main/flight/pid.c

```diff
@@ -460,22 +460,19 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
             // reset ITerm, since accumulated error before crash is now meaningless
             // and ITerm windup during crash recovery can be extreme, especially on yaw axis
             axisPID_I[axis] = 0.0f;
-            if (cmpTimeUs(currentTimeUs, crashDetectedAtUs) > crashTimeLimitUs
+            if (cmpTimeUs(currentTimeUs, crashDetectedAtUs) > crashTimeLimitUs || !ARMING_FLAG(ARMED)
```

Comment:
The addition of `|| !ARMING_FLAG(ARMED)` is the only thing that is needed to fix this bug - the rest of the changes is wrong and has to be reverted.